### PR TITLE
feat(docs): FRMWK-50 - Add Collapsible Layout page

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-50

* [x] Dev LGTM
* [x] Dev LGTM
* [x] Design LGTM

Screenshot:
collapsible elements encoreui

### CODE REVIEW

File: demo/layout/collapsible.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<rx-page title="'Collapsible Elements'" class="styleguide">
+
+    <h3> Collaspible Box </h3>
+    <p>
+        The rxCollapse element can be used to display and hide content. it can be
```
Comment (parlarjb) [inline demo/layout/collapsible.html]: Capital `i` on "it"

Comment (bry4n) [inline demo/layout/collapsible.html]: good catch :+1:

File: demo/demo.js [hunk 2]
```diff
@@ -110,6 +111,13 @@ angular.module('demoApp', ['encore.ui', 'ngRoute'])
                 name: 'Modals'
             }])
         })
+        /* Layout (flexgrid) */
```
Comment (CITguy) [inline demo/demo.js]: Please update to `/* Layouts */`

File: demo/layout/collapsible/see-more-see-less.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	        <dd>Lorem ipsum dolor sit amet</dd>
+	        <dt>ID:</dt>
+	        <dd>1aa2bfa9-de8d-42f7-9f6d-e6437855b36e</dd>
+	        <dt>Region:</dt>
+	        <dd>ORD</dd>
+	        <dt>Created:</dt>
+	        <dd>December 2, 2014 @ 14:28</dd>
+        </rx-collapse>
+    </dl>
+</div>
```
Comment (CITguy) [inline demo/layout/collapsible/see-more-see-less.html]: Please update to use the new [`rxMetadata`](http://rackerlabs.github.io/encore-ui/#/components/rxMetadata) format.

Comment (bry4n) [inline demo/layout/collapsible/see-more-see-less.html]: ah forget about new `rxMetadata` format. It was copy & paste from `rxCollapse` directive page.

**Comment by lfielder** [comment]
Should the ```See more/See less``` section be using the new metadata styles?

**Comment by CITguy** [comment]
@lfielder: Yes, I commented yesterday to update to using rxMetadata.

**Comment by CITguy** [comment]
@bry4n, can you please update the screenshot with the rxMetadata update?

**Comment by CITguy** [comment]
Dev LGTM

**Comment by bry4n** [comment]
Screenshot is updated

**Comment by CITguy** [comment]
Thank you!

**Comment by glynnis** [comment]
@bry4n late to the game here, but there's some information on the component page for rxCollapse that I think also belongs here, namely these additional points for the "See More/See Less" section:

```
- This pattern was developed for areas displaying metadata that may be short on screen real estate, as a way to hide data on load that is not as important to the user in the context where they are presented.
- This pattern is not very responsive-friendly, since as browser width decreases, columns will wrap. As columns wrap, the "See More" rxCollapse elements get lost in the new context, which is bad for user experience.
- To avoid the problem described above, "See More" rxCollapse elements should only be used at the end of the final column present on the page, so that when the columns wrap via flexbox, "See More" is always last and doesn't get lost in between metadata key/value pairs.
```

Once those are added, this has a design LGTM :+2:

**Comment by bry4n** [comment]
@glynnis those are added. screenshot is updated. :ok_hand:

**Comment by glynnis** [comment]
Thanks @bry4n! :cake: 

Design: LGTM

**Comment by halkon** [comment]
LGTM

**Pull request merged**